test/e2e/data/kubetest/conformance-fast.yaml

```diff
@@ -2,7 +2,7 @@ ginkgo.focus: \[Conformance\]
 ginkgo.skip: \[Serial\]
 disable-log-dump: true
 ginkgo.progress: true
-ginkgo.slowSpecThreshold: 120.0
+ginkgo.slow-spec-threshold: 120s
```

Comment:
we have ginkgo v2 specific files for k8s versions that run with v2 https://github.com/kubernetes-sigs/cluster-api-provider-azure/tree/0ba78caed74d57737fd41434f819e73160f18ce5/test/e2e/data/kubetest, did you mean to fix those?

Reply:
I saw a warning about `--slowSpecThreshold` had changed to `--slow-spec-threshold` and grepped about trying to find it, which implied this change. But this actually caused an error because `--gingko.slow-spec-threshold` isn't understood, so I reverted it and will follow up on the warning in a different PR.